executor/ddl.go

```diff
@@ -251,9 +253,26 @@ func isSystemTable(schema, table string) bool {
 	return false
 }
 
+type objectType int
+
+const (
+	tableObject objectType = iota
+	viewObject
+	sequenceObject
+)
+
 func (e *DDLExec) executeDropTableOrView(s *ast.DropTableStmt) error {
```

Comment:
Please separate this function to `executeDropTable` and `executeDropView`(just what you did for Sequences).

Both of them can simply call `dropTableViewSequence`

Reply:
Got it, addressed.